lib/service/service.go

```diff
@@ -3988,9 +3988,17 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 	transportpb.RegisterTransportServiceServer(sshGRPCServer, transportService)
 
 	process.RegisterCriticalFunc("proxy.ssh", func() error {
+		// If a ssh proxy address is specified provide the address and specifics
+		sshListenerAddr := listeners.ssh.Addr().String()
+		sshListenerAddrDetails := sshListenerAddr
+		if cfg.Proxy.SSHAddr.Addr != "" {
+			sshListenerAddr = cfg.Proxy.SSHAddr.Addr
+			// will include address and type (tcp)
+			sshListenerAddrDetails = fmt.Sprintf("%v", cfg.Proxy.SSHAddr)
+		}
 		utils.Consolef(cfg.Console, log, teleport.ComponentProxy, "SSH proxy service %s:%s is starting on %v.",
```

Comment:
I think it's about time we remove `Consolef` - it's only used in a small handful of places and mostly just results in duplicated log messages.

Reply:
removed